<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputPage</class>
 <widget class="QFrame" name="InputPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border-image: url(:/image/icon_num_btn.png);
}

QPushButton:hover, QPushButton:pressed{
	color: rgb(255, 255, 255);
	border-image: url(:/image/icon_num_btn_hover.png);
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>247</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="mpTimeOutLab">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>246</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>59</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="mpBtnRegister">
     <property name="maximumSize">
      <size>
       <width>148</width>
       <height>148</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border-image: url(:/image/icon_register_btn.png);
}

QPushButton:hover, QPushButton:pressed{
	color: rgb(255, 255, 255);
	border-image: url(:/image/icon_register_btn_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="mpSpaceLab">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>108</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>108</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="3" column="0">
        <widget class="QPushButton" name="mpBtnClear">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border-image: url(:/image/icon_clear_btn.png);
}

QPushButton:hover, QPushButton:pressed{
	color: rgb(255, 255, 255);
	border-image: url(:/image/icon_clear_btn_hover.png);
}</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="mpBtn7">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="mpBtn2">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="mpBtn1">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="mpBtn8">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="mpBtn9">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="mpBtn4">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="mpBtn3">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="mpBtnEntry">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border-image: url(:/image/icon_entry_btn.png);
}

QPushButton:hover, QPushButton:pressed{
	color: rgb(255, 255, 255);
	border-image: url(:/image/icon_entry_btn_hover.png);
}</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="mpBtn5">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="mpBtn6">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="mpBtn0">
         <property name="minimumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>148</width>
           <height>148</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="mpInputContent">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>108</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
